project/AppDependencies.scala

```diff
@@ -6,17 +6,16 @@ private object AppDependencies {
 
   val compile = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-play-26"             % "1.8.0",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-27"    % "2.21.0",
     "uk.gov.hmrc" %% "simple-reactivemongo"          % "7.30.0-play-26",
     "uk.gov.hmrc" %% "logback-json-logger"           % "4.8.0",
-    "uk.gov.hmrc" %% "govuk-template"                % "5.55.0-play-26",
-    "uk.gov.hmrc" %% "play-health"                   % "3.15.0-play-26",
-    "uk.gov.hmrc" %% "play-ui"                       % "8.11.0-play-26",
-    "uk.gov.hmrc" %% "http-caching-client"           % "9.1.0-play-26",
+    "uk.gov.hmrc" %% "govuk-template"                % "5.55.0-play-27",
+    "uk.gov.hmrc" %% "play-health"                   % "3.15.0-play-27",
+    "uk.gov.hmrc" %% "play-ui"                       % "8.11.0-play-27",
+    "uk.gov.hmrc" %% "http-caching-client"           % "9.1.0-play-27",
     "uk.gov.hmrc" %% "play-conditional-form-mapping" % "1.2.0-play-26",
-    "uk.gov.hmrc" %% "play-language"                 % "4.3.0-play-26",
-    "uk.gov.hmrc" %% "play-partials"                 % "6.11.0-play-26",
-    "uk.gov.hmrc" %% "domain"                        % "5.9.0-play-26"
+    "uk.gov.hmrc" %% "play-language"                 % "4.3.0-play-27",
+    "uk.gov.hmrc" %% "play-partials"                 % "6.11.0-play-27",
```

Comment:
comma needs to go

Reply:
Not actually true in newest Scala/Play but I'd still like it gone too 😬